modules/installation-initializing.adoc

```diff
@@ -196,8 +206,60 @@ compute:
 <1> Set to `0`.
 endif::[]
 
+ifndef::restricted[]
 . Modify the `install-config.yaml` file. You can find more information about
 the available parameters in the *Installation configuration parameters* section.
+endif::restricted[]
+
+ifdef::osp+restricted[]
+. In `install-config.yaml`, set the value of `platform.openstack.clusterOSImage` to the image location or name. For example:
++
+[source,yaml]
+----
+platform:
+  openstack:
+      clusterOSImage: http://mirror.example.com/images/rhcos-43.81.201912131630.0-openstack.x86_64.qcow2.gz?sha256=ffebbd68e8a1f2a245ca19522c16c86f67f9ac8e4e0c1f0a812b068b16f7265d
+----
+
+. Edit the `install-config.yaml` file to provide the additional information that
+is required for an installation in a restricted network.
+.. Update the `pullSecret` value to contain the authentication information for
+your registry:
++
+----
+pullSecret: '{"auths":{"<bastion_host_name>:5000": {"auth": "<credentials>","email": "you@example.com"}}}'
+----
++
+For `bastion_host_name`, specify the registry domain name
+that you specified in the certificate for your mirror registry, and for
+`<credentials>`, specify the base64-encoded user name and password for
+your mirror registry.
+.. Add the `additionalTrustBundle` parameter and value. The value must be the contents of the certificate file that you used for your mirror registry, which can be an exiting, trusted certificate authority or the self-signed certificate that you generated for the mirror registry.
++
+----
+additionalTrustBundle: |
+  -----BEGIN CERTIFICATE-----
+  ZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZZ
+  -----END CERTIFICATE-----
+----
+.. Add the image content resources:
++
+----
+imageContentSources:
+- mirrors:
+  - <bastion_host_name>:5000/<repo_name>/release
+  source: quay.io/openshift-release-dev/ocp-release
+- mirrors:
+  - <bastion_host_name>:5000/<repo_name>/release
+  source: registry.svc.ci.openshift.org/ocp/release
+----
++
+Use the `imageContentSources` section from the output of the command to
+mirror the repository.
+
+. Make any other modifications to the `install-config.yaml` file that you require. You can find more information about
+the available parameters in the *Installation configuration parameters* section.
```

Comment:
I see that it's formatted this way elsewhere in the file, so it's okay to leave it, but just pointing out that "Installation configuration parameters" should be in italics, not bold.